public/media_engagement/worldbank_plr_sms_traffic.js

```diff
@@ -45,10 +45,18 @@ firebase.auth().onAuthStateChanged(function (user) {
 
 const TIMEFRAME_WEEK = 7;
 const TIMEFRAME_MONTH = 30;
-var chartTimeUnit = "1day";
+const EXTEND_X_AXIS_BY_HOURS = 32
+var chartTimeUnit = "10min";
 var isYLimitReceivedManuallySet = false;
 var isYLimitSentManuallySet = false;
 
+// Date objects inherit from Date.prototype
+// Add the addHours property to the date object
+Date.prototype.addHours= function(h){
+    this.setHours(this.getHours()+h);
+    return this;
```

Comment:
resolved

Reply:
I have written another function to avoid this, here: https://github.com/AfricasVoices/Operations-Dashboard/pull/104/commits/7096dda0f36484443029b955ae713b66b14b0c8d